plugins/database/mysql/mysql_test.go

```diff
@@ -44,6 +45,40 @@ func TestMySQL_Initialize(t *testing.T) {
 	}
 }
 
+// TestMySQL_Initialize_CloudGCP validates the proper initialization of a MySQL backend pointing
+// to a GCP CloudSQL MySQL instance. This expects some external setup (exact TBD)
+func TestMySQL_Initialize_CloudGCP(t *testing.T) {
+	// hardcode data for now
+	url := os.Getenv("CONN_URL")
```

Comment:
In TestInitializeCloudSQL we are using CONNECTION_URL. Is this intentionally different?

Reply:
Updated to be uniform across tests